### PR TITLE
Pension pot values

### PR DESCRIPTION
**Pull request by andrewgarner**

### CODE REVIEW

File: app/templates/output_document.html.erb [hunk 1]
```diff
@@ -162,7 +162,16 @@
...
-    <p class="data"><%= @appointment_summary.value_of_pension_pots %></p>
+    <p class="data">
+      <% if @appointment_summary.value_of_pension_pots.blank? %>
+        Unknown
+      <% elsif @appointment_summary.upper_value_of_pension_pots.blank? %>
+        <%= number_to_currency @appointment_summary.value_of_pension_pots, precision: 0 %>
+      <% else %>
+        <%= number_to_currency(@appointment_summary.value_of_pension_pots, precision: 0) + ' - ' +
+                number_to_currency(@appointment_summary.upper_value_of_pension_pots, precision: 0) %>
+      <% end %>
```
Comment (gjvis) [inline app/templates/output_document.html.erb]: Can we move all of this logic to a presenter?

Comment (andrewgarner) [inline app/templates/output_document.html.erb]: Yep, but if rather do that when we come to render a preview. This was just something quick and dirty to keep moving.



Regards,


Andrew Garner

Guidance Guarantee Programme
HM Treasury

+44 7971 232 140

Comment (gjvis) [inline app/templates/output_document.html.erb]: :+1:

**Comment by gjvis** [comment]
:ship:

**Pull request merged**